### ISSUE TITLE
Скласти таймлайн

### ISSUE THREAD

**Issue by explicity** [explicity/homepage#5]
Скласти список всіх (важливих) життєвих подій, визначних дат, досягнень, періодів, що дали приємний і неприємний досвід, тощо. В цьому конкретному випадку довгих списків не буває, натомість бувають погано згорнуті, тому пакуємо сюди все підряд, а про "лишні" події можна буде промовчати пізніше. Список повинен бути систематизованим і посортованим в такий спосіб, щоб читач швидко зрозумів "паттерн" і міг легко орієнтуватись в просторі і часі цього резюме.

**Comment by explicity** [comment]
**DAENERYS TARGARYEN**

**EXPERIENCE**
**SEASON 8**
* Can't say much yet, but she is totally the Queen.

**SEASON 7**
* Arrives at Dragonstone.  Finally, she’s home.  “Shall we begin?”
* Invites Jon Snow to Dragonstone… to bend the knee.  But he won't do it. Because of him looses one dragon. Fall in love with him. Does it even make a sense?
* Hooks up with Jon in his fur blanket cabin on the way home, while Sam and Bran talk about how she’s his aunt.  Hot.  No, gross. 

**SEASON 6**
* Negotiation with the masters of Slavers Bay.  She’s talking surrender, Drogon drops down off the top of the pyramid in the distance.  “We’re here to discuss your surrender.  Not mine.”  BOSS.  She takes all of the ships she doesn’t burn.  One master is left to tell the tale.  All of Slavers Bay is hers again.  So. Good.
* Theon and Yara come to join forces.  “We are not our fathers.”  She agrees to an alliance if they stay in the Iron Islands and stop raiding.  Agreed.  Queens.  
* Names Tyrion Hand of the Queen.  Sails to Westeros.  

**SEASON 5**
* Tries to visit her dragons in the dungeon.  She misses them, and she’s not herself when they’re tied up.  But they’re bigger and badder, and they don’t really recognize mommy. 
* Meets Tyrion. After much talking, wine and a few threats, she decides to keep him as an advisor.  “I’m not going to stop the wheel.  I’m going to break the wheel.”
* She’s trapped in the center of the great pit. Enter Drogon.  Do you think Drogon doesn’t know when she’s in trouble?  He does.  He comes in and starts f@#8*ing everyone up. But, he’s taking a lot of spears and she needs to get him out of there before he dies trying to defend her.  So… let’s ride! They disappear over the horizon.  Yes yes yes. 

**SEASON 4**
* Dragons are bigger than people for the first time.  Start with some nice dragon time by the sea.  That is, until they start fighting over a lamb carcass and it becomes clear about to cause some sh*@!
* Going to Meereen. The city is hers, and she means to settle the score.  She crucifies the masters, numbered as many as the children she passed on the way.  “I will answer injustice with justice.”
* Queen stuff is hard.  Goats are being dragon-torched. Then Drogon kills a toddler.  She can’t take it.  She chains Viserion and Rhaegal.  Drogon is nowhere to be found. 

**SEASON 3**
* Sails to Astapor.  The dragons are bigger.  “Not big enough.” She needs an army.
* Meets the master of the Unsullied.  Offers him a dragon in return for all 8,000 Unsullied. Done.  She takes Missandei as a ‘gift’, because she recognizes the awesome.  Immediately frees her.
* The exchange.  The whip that commands the Unsullied is handed over.  Drogon is handed over.  Once the deal is done, she breaks out the Valeryian.  Boss.  She commands the Unsullied to kill the masters, spare the innocents.  Then, Dracarys!  Dragon burns his new master to ash.  THIS IS ONE OF THE BEST SCENES IN THE SERIES.
* Kill masters in Yunkai, freed slaves open the gates of Yunkai chanting “Mhysa”, and she’s in. 

**SEASON 2**
* She is in trouble. Again. Her people are starving. Horses are dying, including her beautiful white horse… Her dragons won’t eat.  She only knows to give them meat.  She doesn’t know the secret is fire. The secret is always fire.  At some point she figures that out and starts commanding ‘Dracarys’ so they roast their food.
* A rider finds a new town - Qarth, but gates are closed and they're denied entry.
* Enters Qarth. It's a party (actually Dracarys kills some of the masters, it's not a party).

**SEASON 1**
* Daenerys’ only surviving family member (or so we think), Viserys, her brother, is a coward and a monster.  He sells her to Khal Drogo for a chance to take a Dothraki Khalasar to Westeros and win the Iron Throne.  Also, he’s gross and full of bad touch moments.
* Gets a Dothraki wedding.  She’s gifted dragon eggs, which everyone just thinks are pretty rocks.  The Dothraki are a brutal people.  People die.  Women are raped.
* Gets pregnant immediately.  
* Eats horse heart… throws it up… eats THAT. Becomes Dothraki.  
* Baby dies. Drogo dies too. Even Viserys dies! Things are so bad.  Blood magic is the worst. 
* But wait! Morning comes and Daenerys emerges from the ashes with three baby dragons!  She is the unburnt.  The mother of dragons.  We know who the real dragon is. 

**PERSONAL LIFE**
Should we really talk about her nephew here?
